src/components/Trilemma/Trilemma.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, ChakraProvider } from "@chakra-ui/react"
+import { Meta, StoryObj } from "@storybook/react"
+
+import Trilemma from "."
+
+const meta = {
+  title: "Molecules / Trilemma",
+  component: Trilemma,
+  parameters: {
+    layout: "fullscreen",
+  },
+  decorators: [
+    (Story) => (
+      <ChakraProvider>
+        <Box
+          w="100vw"
+          h="100vh"
```

Comment:
<img width="1489" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/3a694215-f042-4e4f-9271-5b508d027603">

Not 100% sure we want to use `100vh` here since it makes it behave differently than in production.

Reply:
<img width="418" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/027ee239-63e6-4f08-84f7-fb1a69dc3ba6">

Similar on mobile, where it also cuts off the title...

And in this screenshot we're lacking any padding on the edges as well, compared to production:

<img width="356" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/99cd48bb-da7b-42d3-be39-30d7a7d83936">